core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.ts

```diff
@@ -213,13 +209,11 @@ export class StoryEditorNavbarComponent implements OnInit {
 
   selectMainTab(): void {
     this.activeTab = this.EDITOR;
-    this.storyEditorNavigationService.navigateToStoryEditor();
     this.showNavigationOptions = false;
   }
 
   selectPreviewTab(): void {
     this.activeTab = this.PREVIEW;
-    this.storyEditorNavigationService.navigateToStoryPreviewTab();
```

Comment:
Ditto

Reply:
This function is doing nothing/empty here.

Reply:
fixed this and tested on mobile view working fine now.

Reply:
https://user-images.githubusercontent.com/54100702/121724665-cf696d00-cb05-11eb-8dc7-80c73b6e742c.mp4